components/forms/SearchForm.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import styles from "../../styles/SearchBar.module.css";
+import SearchBar from "../home/SearchBar";
+
+export default class SearchForm extends React.Component {
```

Comment:
I created this component as well, doesn't have much to do with the include_saga task